tron/serialize/runstate/dynamodb_state_store.py

```diff
@@ -194,8 +197,14 @@ def __setitem__(self, key: str, val: bytes) -> None:
                     count += 1
                     if count > 3:
                         raise e
+                    log.warning(f'Got error while saving, trying again: {repr(e)}')
+        timer(
+            name='tron.dynamodb.setitem',
+            delta=time.time() - start,
+        )
 
     def _delete_item(self, key: str) -> None:
+        start = time.time()
```

Comment:
i think we should wrap the whole method in try/except and report timer in finally clause